backend/baseclass.pm

```diff
@@ -296,17 +291,9 @@ sub run_capture_loop {
 sub check_select_rate {
     my ($buckets, $wait_time_limit, $hits_limit, $id) = @_;
 
-    my $time        = gettimeofday;
+    my $time = gettimeofday;
+    $buckets->{TIME} //= $time;
```

Comment:
In the old code `$lower_limit` will be set to `$buckets->{TIME}` if it is set.
I don't see that in the new code.

Reply:
right. How about

```suggestion
    my $lower_limit = $buckets->{TIME} //= $time;
```